### PR TITLE
First commit in Programmers-Community

### PR DESCRIPTION
**Pull request by haardikdharma10**

### CODE REVIEW

**Comment by haardikdharma10** [comment]
Changes made!

**Comment by haardikdharma10** [comment]
Done, please check.

File: Basic/Check Whether a Number is Even or Odd/SolutionByHaardik.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        Scanner in = new Scanner(System.in);
+        System.out.println("Enter a number");
+        int n = in.nextInt();
+        System.out.println(isEven(n));
+    }
+
+
+    public static String isEven(int n) {
+        if (n % 2 == 0)
+            return "The number is even";
```
Comment (shoaibrayeen) [inline Basic/Check Whether a Number is Even or Odd/SolutionByHaardik.java:14]: You can't use % operator as mentioned in readMe.

File: Basic/Calculate Factorial of A Number/SolutionByHaardik.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        int number = in.nextInt();
+        if(number<0){
+            System.out.println("Factorial is not defined for negative numbers!");
+        }
+        else {
+            System.out.println("The factorial of " + number + " is " + factorial(number));
+        }
+    }
+
+    public static int factorial(int n){
```
Comment (shoaibrayeen) [inline Basic/Calculate Factorial of A Number/SolutionByHaardik.java:16]: The return type should be BigInt.

File: Data Structure/Array Or Vector/Sum of Elements in An Array/SolutionByHaardik.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        int [] arr = new int [num];
+        System.out.println("Enter the elements of array");
+        for (int i = 0; i<num; i++){
+            arr[i] = input.nextInt();
+        }
+        for (int number : arr) {
+            sum = sum + number;
+        }
+            System.out.println("The sum of the elements is: " +sum);
+
```
Comment (shoaibrayeen) [inline Data Structure/Array Or Vector/Sum of Elements in An Array/SolutionByHaardik.java:18]: no extra line should be there.

File: Basic/Check Whether a Number is Even or Odd/SolutionByHaardik.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        else
+            return "The number is odd";
+    }
+}
+
+
```
Comment (shoaibrayeen) [inline Basic/Check Whether a Number is Even or Odd/SolutionByHaardik.java:20]: remove all extra lines.

**Pull request closed without merging**